.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!-- Provide a brief description of the PR -->
+
+#### Related PRs / Issues
+
+<!-- List any related issues here -->
+
+#### API Impact
+
+<!-- How would this affect the current API? Does this add, extend, deprecate, remove, or break any existing API? -->
+
+#### Verilog / AGFI Compatability
+
+<!-- Does this change any generated Verilog, or the simulator memory map?  -->
+
+#### Release Notes
+
+<!-- You'll copy text from here to the dev-to-master PR.  -->
```

Comment:
I can see why it might be useful to have a separate section for this but I typically try to make the title of my PR usable as the release notes so I can copy paste it in there.